DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/_OtherCompletionCriteriaInputs.cshtml

```diff
@@ -0,0 +1,18 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@model EditCourseDetailsViewModel
+
+<vc:numeric-input asp-for="@nameof(Model.TutCompletionThreshold)"
+                  label="Required learning percentage"
+                  populate-with-current-value="true"
+                  type="text"
+                  hint-text=""
+                  css-class="nhsuk-input nhsuk-input--width-10 nhsuk-u-margin-bottom-2" />

```

Comment:
Duplicate nhsuk-input class on the input. Same for the other component in the file

Reply:
removed